_migration-assistant/migration-phases/planning-your-migration/adjusting-settings-during-migration.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Adjusting settings during migration
+nav_order: 60
+parent: Planning your migration
+grand_parent: Migration phases
+redirect_from:
+  - /migration-assistant/migration-phases/adjusting-settings-during-migration/
+---
+
+# Adjusting settings during migration
+
+During a migration there is an opportunity change names and paths of index and data structures that are not modifiable after they are declared. This page provides a cookbook of different scenarios and templates that can be used to make these adjustments.
+
+### Rename an index
+
+> [!NOTE]  
+> We recommend creating an alias to an index instead of renaming them, if an alias is created on the source cluster the migration process will copy the alias over as well. See create an alias [documentation](https://opensearch.org/docs/latest/api-reference/index-apis/update-alias/).
+
+#### Migration with a snapshot
+
+1. Navigate to the bootstrap box and open the `cdk.context.json` with vim
+2. Add/Update the key `reindexFromSnapshotExtraArgs` to include `--doc-transformer-config-file /shared-logs-output/rfs-transform.json`
+3. Redeploy the Migration Assistant 
+4. Navigate on to the Migration Assistant console
+5. Create a file with `vim /shared-logs-output/rfs-transform.json`
```

Comment:
We shouldn't recommend to users that they WRITE a transform in a shared log.  It's arguable if that volume should be writable by the migration console.  Same argument for the other file below.

Reply:
Thanks - what is an alternative way to accomplish this?

Reply:
If you have a file, here's how you can convert it into json and then base64
```
jq -n --rawfile script1 ~/Downloads/es-load-test.py '{"fileContent": $script1}'| base64
```
That could then be pushed into the extra args for the applications ("--transformer-config-base64 ...").  Notice that we'll need the json to match what the JsonJSTransformerProvider expects in the PR ([see here](https://github.com/opensearch-project/opensearch-migrations/pull/1207/files#diff-2223bbb5e0fd42b5923d57314dc4741f98d8ed14136b23517995cc3d93cea243)).